dvc/logger.py

```diff
@@ -200,5 +200,6 @@ def setup(level=logging.INFO):
                     ],
                 },
             },
+            "disable_existing_loggers": False,
```

Comment:
How noisy is `gitpython` (or, similar dependencies)  on logging, @efiop? We might need to explicitly disable those for good, otherwise, it might mess with cli commands(?).

We need to reset loglevel of `moto` to `logging.CRITICAL`: https://github.com/spulec/moto/issues/2535, or just disable other loggers during test.

Reply:
@skshetry They are indeed fairly noisy. Maybe let's `disable_existing_loggers` in CLI and not touch anything in API? We do a similar thingy with some warnings in `dvc/__init__.py`, but that is rather incorrect. We should disable logs and warnings in `dvc/main.py::main()` most likely. What do you think?

Reply:
Yeah, handling in `main.py` is better. The question is, should we be explicitly disable other loggers there or disable every loggers except of DVC in the `main.py`?

Reply:
@skshetry I would just disable everything except DVC, to avoid chasing every weird dependency logger out there.

Reply:
Just a note regarding adjustment of tests: If we disable all the loggers in `main.py`, tests where we do not work with `main()` such as `Repo.push`, etc will have all other loggers enabled. So, we might need to disable some loggers on the tests as well (such as `moto` which is making a test fail at the moment).